test/v1/connectors/BaseDownstreamConnectorSpec.scala

```diff
@@ -257,4 +272,99 @@ class BaseDownstreamConnectorSpec extends ConnectorSpec {
     }
   }
 
+  "for TYS IFS" when {
```

Comment:
Is this a new test? Just wondering if it ought to be copied into the other API codebases as well?